exports/contexts/obo.epm.json

```diff
@@ -26587,7 +26532,6 @@
             "http://bioregistry.io/string:",
             "http://bioregistry.io/swiss-model:",
             "http://bioregistry.io/topfind:",
-            "http://bioregistry.io/uniprot/swiss-prot:",
```

Comment:
I assume this is removed because of another reason!

Reply:
the prefix synonym `uniprot/swiss-prot` is not W3C-compliant since it has a slash in it, so it got removed